src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name, hidden = false) {
+    super(health, name);
+
+    this.hidden = hidden;
+  }
```

Comment:
You are trying to provide value in the func. constructor that it does not expect. You don't need to provide `health`.
![image](https://user-images.githubusercontent.com/108505176/187025647-d8693bab-3026-4635-9995-4db1fa63bf06.png)